app/views/shared/_header.html.haml

```diff
@@ -120,3 +120,6 @@
             %li= link_to t(:account_settings), edit_person_path(current_user), :class => 'profile_link'
             %li.divider{role: "separator"}
             %li.signout= link_to t(:sign_out), logout_path
+  #initiative.navtabs
```

Comment:
Can you wrap this in something like `if @responsive`? This doesn't need to be on every page on the site, even if its hidden. I'd also appreciate a more descriptive name, like `#supporter-branding` or something.